services/sqs/src/test/java/software/amazon/awssdk/services/sqs/SqsBatchFunctionsTest.java

```diff
@@ -0,0 +1,278 @@
+import static software.amazon.awssdk.services.sqs.internal.batchmanager.SqsBatchFunctions.deleteMessageBatchKeyMapper;
+import static software.amazon.awssdk.services.sqs.internal.batchmanager.SqsBatchFunctions.deleteMessageResponseMapper;
+import static software.amazon.awssdk.services.sqs.internal.batchmanager.SqsBatchFunctions.sendMessageBatchKeyMapper;
+import static software.amazon.awssdk.services.sqs.internal.batchmanager.SqsBatchFunctions.sendMessageResponseMapper;
+
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import software.amazon.awssdk.awscore.AwsRequestOverrideConfiguration;
+import software.amazon.awssdk.core.internal.batchmanager.IdentifiableMessage;
+import software.amazon.awssdk.services.sqs.model.ChangeMessageVisibilityBatchResponse;
+import software.amazon.awssdk.services.sqs.model.ChangeMessageVisibilityBatchResultEntry;
+import software.amazon.awssdk.services.sqs.model.ChangeMessageVisibilityRequest;
+import software.amazon.awssdk.services.sqs.model.ChangeMessageVisibilityResponse;
+import software.amazon.awssdk.services.sqs.model.DeleteMessageBatchResponse;
+import software.amazon.awssdk.services.sqs.model.DeleteMessageBatchResultEntry;
+import software.amazon.awssdk.services.sqs.model.DeleteMessageRequest;
+import software.amazon.awssdk.services.sqs.model.DeleteMessageResponse;
+import software.amazon.awssdk.services.sqs.model.SendMessageBatchResponse;
+import software.amazon.awssdk.services.sqs.model.SendMessageBatchResultEntry;
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest;
+import software.amazon.awssdk.services.sqs.model.SendMessageResponse;
+import software.amazon.awssdk.utils.BinaryUtils;
+import software.amazon.awssdk.utils.Md5Utils;
+
+// TODO: Will refactor code in this test file for async tests.
+public class SqsBatchFunctionsTest {
+
+    @Test
+    public void sendMessageResponseMapper_mapResponsesCorrectly() {
+        String id1 = "1";
+        String id2 = "2";
+        String messageBody1 = getMd5Hash("1");
+        String messageBody2 = getMd5Hash("2");
+        SendMessageBatchResultEntry entry1 = createSendMessageBatchEntry(id1, messageBody1);
+        SendMessageBatchResultEntry entry2 = createSendMessageBatchEntry(id2, messageBody2);
+        SendMessageBatchResponse batchResponse = SendMessageBatchResponse.builder()
+                                                                         .successful(entry1, entry2)
+                                                                         .build();
+        List<IdentifiableMessage<SendMessageResponse>> mappedResponses =
+            sendMessageResponseMapper().mapBatchResponse(batchResponse);
+
+        IdentifiableMessage<SendMessageResponse> response1 = mappedResponses.get(0);
+        IdentifiableMessage<SendMessageResponse> response2 = mappedResponses.get(1);
+        Assert.assertEquals(id1, response1.id());
```

Comment:
nit, assertj library is preferred because of its fluent APIs and better assertion error messages https://assertj.github.io/doc/

Reply:
Should I refactor all test files to use assertJ library then?

Reply:
No, this is just nitpicking. I don't think there are complicated assertions anyway. Let's use assertJ going forward.

Reply:
Refactored all test files to use assertJ library